data/json/recipes/recipe_others.json

```diff
@@ -3883,7 +3883,7 @@
       { "id": "CONTAIN", "level": 1 }
     ],
     "tools": [ [ [ "surface_heat", 10, "LIST" ] ] ],
-    "components": [ [ [ "water_clean", 2 ], [ "water", 2 ] ], [ [ "lye_powder", 10 ], [ "aspirin", 10 ] ], [ [ "cured_hide", 1 ] ] ]
+    "components": [ [ [ "water_clean", 1 ], [ "water", 1 ] ], [ [ "lye_powder", 10 ], [ "aspirin", 10 ] ], [ [ "cured_hide", 6 ] ] ]
```

Comment:
Unless I'm missing something, the amount of water required here should remain the same.

Reply:
No, you're not missing anything. Changing this was not the intent. Reverted, thanks!